logrotate.c

```diff
@@ -31,13 +31,13 @@
 #include "logrotate.h"
 
 static void *prev_context;
-#ifdef WITH_SELINUX
+#ifdef HAVE_LIBSELINUX
```

Comment:
While this change is definitely an improvement over status quo, I would prefer to keep using ```WITH_SELINUX``` (and ```WITH_ACL```) in code and have it defined by autoconf.  Obviously, ```WITH_SELINUX``` is a more intuitive name of the define in case you have libselinux on your system but compile logrotate without SELinux support.